_toc.yml

```diff
@@ -1,28 +1,22 @@
-- title: Home
-  file: intro
-
-- file: authoring/index
-  expand_sections: true
+file: intro
+sections:
+- file: README
+- file: open-source-software-peer-review/index
   sections:
-    - file: authoring/overview
-    - file: authoring/testing
-    - file: authoring/collaboration
-    - file: authoring/release
-
-- file: process/intro
-  expand_sections: true
+  - file: open-source-software-peer-review/code-of-conduct
+  - file: open-source-software-peer-review/aims-and-scope
+  - file: open-source-software-peer-review/policies-and-guidelines
+- file: authoring/index
   sections:
-    - file: process/aims_scope
-    - file: process/guidelines
-    - file: process/coc
-
-- file: submissions/intro
-  expand_sections: true
+  - file: authoring/collaboration
+  - file: authoring/overview
+  - file: authoring/testing
+  - file: authoring/release
+- file: submissions/editor_guide
   sections:
   - file: submissions/author_guide
+  - file: submissions/intro
   - file: submissions/reviewer_guide
-  - file: submissions/editor_guide
-
 - file: appendices/templates
-- file: appendices/glossary
-
+  sections:
```

Comment:
attempting to force the order. but it seems to want to order itself when i build locally.